### PR TITLE
deployment update

### PR DESCRIPTION
**Pull request by yiyione**
Update Dockerfile, sequalize model and change get items function to use RestAPI

### CODE REVIEW

File: rest_server/src/models/marketplace_item.js [hunk 1]
```diff
@@ -37,23 +47,28 @@ class MarketplaceItem {
...
+      async (name, author, category, status, type) => {
+        const filterStatement = {};
+        if (name) {
+          filterStatement.name = name;
+        }
+        if (author) {
+          filterStatement.author = author;
+        }
+        if (category) {
+          filterStatement.category = category;
```
Comment (debuggy) [inline rest_server/src/models/marketplace_item.js:60]: categories is array type, the logic might be not suitable for the new type. Maybe you could delete these related queries that not used now. like category and status

File: examples/constants.js [hunk 2]
```diff
@@ -0,0 +1,477 @@
...
+// Licensed under the MIT License.
+
+/* eslint-disable no-template-curly-in-string */
+
+const MARKET_ITEM_LIST = [
+  {
+    id: '0b41d10e-36e5-4e97-95c2-27726dd94f33',
+    name: 'COVID-19 image data collection',
+    author: 'ieee8023',
+    type: 'data',
```
Comment (debuggy) [inline examples/constants.js:11]: Could you add dataUrl and dataType values after the new download feature merged?

File: rest_server/package.json [hunk 3]
```diff
@@ -4,9 +4,6 @@
   "description": "The rest server of openpai marketplace.",
   "author": "Microsoft",
   "license": "MIT",
-  "engines": {
```
Comment (debuggy) [inline rest_server/package.json:7]: what is the requirement of node version now?

File: rest_server/src/models/marketplace_item.js [hunk 4]
```diff
@@ -7,25 +7,35 @@ class MarketplaceItem {
...
+        allowNull: false,
+        defaultValue: 0,
+      },
       starNumber: {
         type: DataTypes.INTEGER,
         allowNull: false,
         defaultValue: 0,
       },
       status: DataTypes.ENUM('pending', 'approved', 'rejected'),
+      createdAt: {
```
Comment (debuggy) [inline rest_server/src/models/marketplace_item.js:34]: the createdAt value is set by rest server or database?

File: rest_server/src/models/marketplace_item.js (hunk 1 above)
Comment (yiyione) [reply rest_server/src/models/marketplace_item.js]: Done.

File: examples/constants.js (hunk 2 above)
Comment (yiyione) [reply examples/constants.js:11]: OK, will add after the download PR merge

File: rest_server/package.json (hunk 3 above)
Comment (yiyione) [reply rest_server/package.json]: The webportal in marketplace require node 10, but the rest-server works well with node 8.
I just forget to add "node": "^10" back.
Fixed.

File: rest_server/src/models/marketplace_item.js (hunk 4 above)
Comment (yiyione) [reply rest_server/src/models/marketplace_item.js:34]: rest server

File: examples/constants.js (hunk 2 above)
Comment (yiyione) [reply examples/constants.js:11]: Done.

**Review by debuggy — APPROVED** [review]

**Pull request merged**